src/partition/repart.c

```diff
@@ -4775,6 +4775,18 @@ static int make_subvolumes_set(
         return 0;
 }
 
+static usec_t epoch_or_utime_omit(void) {
+        uint64_t epoch;
+
+        if (secure_getenv_uint64("SOURCE_DATE_EPOCH", &epoch) >= 0) {
```

Comment:
we nowadays prefer that any unexpected failures result in debug logs, hence maybe change to:

```
r = secure_getenv_uint64(…);
if (r >= 0) {
        …
} else if (r != -ENXIO) 
        log_debug_errno(r, "Failed to parse $SOURCE_DATE_EPOCH, ignoring: %m");
```

or so